app/invocation/invocation.tsx

```diff
@@ -344,6 +347,9 @@ export default class InvocationComponent extends React.Component<Props, State> {
       .then((response) => {
         const runnerExecution = response.execution?.[response.execution.length - 1] ?? undefined;
         this.setState({ runnerExecution });
+      })
+      .catch((e) => {
+        console.error("Failed to fetch runner execution", e);
```

Comment:
want to also throw in a dummy setState here? maybe `this.setState({ runnerExecution: undefined })`

Reply:
Sure, done